wandb/data_types.py

```diff
@@ -932,6 +954,30 @@ def captions(cls, videos):
             return False
 
 
+class Classes(Media):
+    def __init__(self, class_set):
+        self._class_set = class_set
+        # TODO: validate
+
+    def to_json(self, artifact):
+        return {"_type": "classes", "class_set": self._class_set}
+
+
+class JoinedTable(Media):
+    def __init__(self, table1_path, table2_path, join_key):
```

Comment:
Since `Table` is instantiated with a `name` string, it is not necessarily clear that the name is also the path. What do you think about the JoinedTable constructor accepting two wandb.Table objects. This way the user does not need to think about file paths at all here.